aries_cloudagent/anoncreds/did_web_registry/definition.py

```diff
@@ -0,0 +1,10 @@
+"""Version definitions for this protocol."""
```

Comment:
This file is out of place since we're not loading any protocols; the routes.py should cause this package to be loadable as a plugin